projects/tinkerbell/tink/build/create_manifests.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+set -x
+set -o errexit
+set -o pipefail
+
+REPO="$1"
+OUTPUT_DIR="$2"
+ARTIFACTS_PATH="$3"
+TAG="$4"
+IMAGE_REPO="$5"
+IMAGE_TAG="$6"
+GOLANG_VERSION="$7"
+
+MAKE_ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd -P)"
+source "${MAKE_ROOT}/../../../build/lib/common.sh"
+
+build::common::use_go_version ${GOLANG_VERSION}
+
+cd $REPO
+
+npm install prettier --global
```

Comment:
What are they using this for?

Reply:
`prettier` is used at a bunch of places in tink's mk to beautify the generated YAMLs and stuff https://github.com/tinkerbell/tink/blob/main/kube.mk#L63
I don't think it is required or anything since all of these manifests are auto-generated using controller-gen and kustomize so I could patch that out if needed.

Reply:
no i dont think it matters, just dont usually see go projects pulling in anything but other go utils, esp not a npm package!

Reply:
yeah it is weird, prettier only has npm and yarn packages